src/jv.c

```diff
@@ -484,6 +482,36 @@ int jvp_number_cmp(jv a, jv b) {
   }
 }
 
+jv jv_number_add(jv a, jv b) {
+  if (jvp_number_is_literal(a) && jvp_number_is_literal(b)) {
+    decNumber *da = jvp_dec_number_ptr(a);
+    decNumber *db = jvp_dec_number_ptr(b);
+
+    unsigned digits = MAX(da->digits+da->exponent, db->digits+db->exponent) -
+                      MIN(da->exponent, db->exponent) + 1;
+    jvp_literal_number *nlit = jvp_literal_number_alloc(digits);
+    nlit->refcnt = JV_REFCNT_INIT;
+    nlit->literal_data = NULL;
+    nlit->num_double = NAN;
+
+    decContext *ctx = DEC_CONTEXT();
+    decNumberAdd(&nlit->num_decimal, da, db, ctx);
+    decNumberReduce(&nlit->num_decimal, &nlit->num_decimal, ctx);
```

Comment:
If we believe that the operation above (the add) is able to fit the result into our calculated digits then the reduce should not be needed

Reply:
After performing some tests I can tell that `decNumberAdd` will function correctly with our digits calculation and the only thing we should do after is `decNumberTrim`. We shouldn't call `decNumberReduce` as it will convert the number to its minimal scientific form which is not desirable IMO:

Using reduce: 
```61e2 + 00009876543210000000e-2 => 9.87654321061E+13```

Using trim:
```61e2 + 00009876543210000000e-2 => 98765432106100```

Reply:
Yeah, the `decNumberReduce` was meant to address the trailing zero issue- I wasn't concerned about `decNumberAdd` functioning properly. `decNumberTrim` is the best option, I'll swap to that.